### DISCUSSION TITLE
What was BeatBattle.hx for

### DISCUSSION THREAD

**Discussion by AydenTheProgrammer** [FunkinCrew/Funkin#1695] [Q&A]
what was beatbattle.hx for? i was just curious lol!

**Comment by Cheemsandfriends** [comment]
Maybe it was for marking the beats. Lol

**Comment by libauror-a** [comment]
well i deleted it :trollface:

**Comment by Jaduplex** [comment]
it was for after you finish the week but it never had any work done on it
